### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/deployment/index.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/deployment/index.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/deployment/index.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 16ed179f3aac9e479b626ff5902e27470ee8ae40 at: https://1d19be67.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 40363ae45a79b71410f64a2db2b21fd270966ce0 at: https://9ae3708f.fastapitiangolo.pages.dev

File: docs/tr/docs/deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Deployment Stratejileri
+
+Kullanım durumunuza ve kullandığınız araçlara bağlı olarak bir kaç farklı yol izleyebilirsiniz.
+
+Bir dizi araç kombinasyonunu kullanarak kendiniz **bir sunucu yayınlayabilirsiniz**, yayınlama sürecinin bir kısmını sizin için gerçekleştiren bir **bulut hizmeti** veya diğer olası seçenekleri kullanabilirsiniz.
+
+**FastAPI** uygulamasını yayınlarken aklınızda bulundurmanız gereken ana kavramlardan bazılarını size göstereceğim (ancak bunların çoğu diğer web uygulamaları için de geçerlidir).
+
+Sonraki bölümlerde akılda tutulması gereken diğer ayrıntıları ve tekniklerinden bazılarını göreceksiniz. ✨
```
Comment (alperiox) [inline docs/tr/docs/deployment/index.md:21]: Yayinlama tekniklerinden bahsedildigini vurgulayabiliriz:

```suggestion
Sonraki bölümlerde akılda tutulması gereken diğer ayrıntıları ve yayınlama tekniklerinden bazılarını göreceksiniz. ✨
```

File: docs/tr/docs/deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Deployment (Yayınlama)
+
+**FastAPI** uygulamasını deploy etmek oldukça kolaydır.
+
+## Deployment Ne Anlama Gelir?
+
+Bir uygulamayı **deploy** etmek (yayınlamak), uygulamayı **kullanıcılara erişilebilir hale getirmek** için gerekli adımları gerçekleştirmek anlamına gelir.
+
+Bir **Web API** için, bu süreç normalde uygulamayı **uzak bir makineye** yerleştirmeyi, iyi performans, kararlılık vb. özellikler sağlayan bir **sunucu programı** ile **kullanıcılarınızın** uygulamaya etkili ve kesintisiz bir şekilde **erişebilmesini** kapsar.
```
Comment (alperiox) [inline docs/tr/docs/deployment/index.md:9]: Okunusu biraz daha akici oldu sanirim: 

```suggestion
Bir **Web API** için bu süreç normalde uygulamayı **uzak bir makineye** yerleştirmeyi, iyi performans, kararlılık vb. özellikler sağlayan bir **sunucu programı** ile **kullanıcılarınızın** uygulamaya etkili ve kesintisiz bir şekilde **erişebilmesini** kapsar.
```

**Review by alperiox — COMMENTED** [review]
I had some suggestions about the general structure of the translation. Of course, it isn't necessary to change it according to my suggestions. Besides those suggestions, the translation looks nice and clean. Thanks for the hard work!

**Comment by hasansezertasan** [comment]
> I had some suggestions about the general structure of the translation. Of course, it isn't necessary to change it according to my suggestions. Besides those suggestions, the translation looks nice and clean. Thanks for the hard work!

Thank you for the review 🙏, I have committed your change requests and I think it looks far better now 🤓.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 675fcb99662c3018e1364fd2635fc5148864b647 at: https://bcb4d7b0.fastapitiangolo.pages.dev

**Review by alperiox — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2423be69d47aeb960ad8bdf3b6a61529707b2657 at: https://5a3a34f5.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thank you! nice job @hasansezertasan :partying_face: 
Thanks for the review @alperiox :v:

**Pull request merged**